### DISCUSSION TITLE
Native search

### DISCUSSION THREAD

**Discussion by eswanzey** [vanjarosoftware/Vanjaro.Platform#1029] [Ideas]
DNNSharp's SearchBoost module gave us a decent DNN search engine when there wasn't a good alternative. I recently sat in on a webinar where they discussed their future direction, and they are quickly migrating away from offering individual modules.

I hope that core Vanjaro will offer the ability to spider pdf's and customize the search results displays. Of course it will. Those have been two major misses on DNN native search.

**Comment by meetmandeep** [comment]
Vanjaro as a platform will continue to support open architecture for other vendors or community to fill in and add value. Vanjaro as a company will also offer commercial extensions in future. 

Search works on a provider model so it should be easy to switch to another solution.

**Reply by Tychodewaard** [reply]
Is there any info on how to get Search boost working in Vanjaro?
